Typography/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="./index.css" rel="stylesheet">
+    <title>Neon-UI | Typography</title>
+</head>
+
+<body>
+    <div id="typography">
+        <h1 class="mb-1">Typography</h1>
+        <h3>Example of headings:</h3>
+        <p class="text-medium">For Heading we can use h1, h2, h3, h4, h5 elements.</p>
+        <p class="text-medium">Same size can be applied using text classes :- xl, large, medium, small.</p>
+        <div class="typo-container">
+            <h1>This is H1 heading.</h1>
+            <h2>This is H2 heading.</h2>
+            <h3>This is H3 heading.</h3>
+            <h4>This is H4 heading.</h4>
+            <h5>This is H5 heading.</h5>
+        </div>
+
+        <div class="typo-container">
+            <p class="text-xl">Paragraph with class- text-xl</p>
+            <p class="text-large">Paragraph with class- text-large</p>
+            <p class="text-medium">Paragraph with class- text-medium</p>
+            <p class="text-small">Paragraph with class- text-small</p>
+        </div>
+        <!-- <div class="code-snipet-container">
+            
+        </div> -->
```

Comment:
Try avoiding commented code in the PR. It's considered bad practice as suggested by our MA Pranshu bhaiya.

Reply:
Okay.